applications/IgaApplication/custom_processes/output_quadrature_domain_process.cpp

```diff
@@ -44,20 +44,20 @@ void OutputQuadratureDomainProcess::ExecuteBeforeSolutionLoop()
     if (OutputGeometryElements) {
         for (auto element : r_model_part.Elements()) {
             auto integration_point = element.GetGeometry().IntegrationPoints()[0];
-            contents += '[' + element.Id() + ',' + element.GetGeometry().GetGeometryParent(0).Id() + ',[';
+            contents += '[' + element.Id() + ',' + element.GetGeometry().GetGeometryParent(0).Id() + ",[";
```

Comment:
It should be std::to_string(element.Id()) and std::to_string(element.GetGeometry().GetGeometryParent(0).Id()).

Reply:
thanks for the comment. I updated it.